content/en/vendor/redhat/pipelines-install.md

```diff
@@ -19,10 +19,13 @@ Constraints][security-con] for more information.
    ```
 1. Install Tekton Pipelines:
 
+    Because OpenShift uses random user id (and user id range per namespace) for pods, we need to remove the `securityContext.runAsUser` and `securityContext.runAsGroup` from any container from the release.yaml.
+
    ```bash
-   oc apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml
+   curl https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml | yq 'del(.spec.template.spec.containers[].securityContext.runAsUser, .spec.template.spec.containers[].securityContext.runAsGroup)' | oc apply -f -
```

Comment:
Maybe mention explicitly that yq is needed for this, not everyone will have it installed

Reply:
Good point 👼🏼

Reply:
or just used sed and don't make yq as deps

```
curl -s https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml|sed '/runAs/d'|kubectl apply -f-
```